packages/esm-patient-labs-app/src/lab-orders/add-lab-order/add-lab-order.workspace.tsx

```diff
@@ -11,7 +11,7 @@ import {
   type LabOrderBasketItem,
   launchPatientWorkspace,
 } from '@openmrs/esm-patient-common-lib';
-import { TestTypeSearch } from './test-type-search';
+import { TestTypeSearch } from './test-type-search.component';
```

Comment:
Renamed to match our [naming conventions](https://o3-docs.openmrs.org/docs/coding-conventions#naming).